update.sh

```diff
@@ -12,17 +12,20 @@ versions=( "${versions[@]%/}" )
 travisEnv=
 for version in "${versions[@]}"; do
 	travisEnv='\n  - VERSION='"$version$travisEnv"
-	
-	repoBase="http://packages.elasticsearch.org/elasticsearch/$version/debian"
-	if [ "${version%%.*}" -ge 2 ]; then
-		repoBase="http://packages.elasticsearch.org/elasticsearch/${version%%.*}.x/debian"
-	fi
-	
+
+	majorVersion="${version%%.*}"
+	repoBase="http://packages.elasticsearch.org/elasticsearch/${majorVersion}.x/debian"
+	case "$majorVersion" in
+		5) repoBase="https://artifacts.elastic.co/packages/${majorVersion}.x-prerelease/apt" ;;
```

Comment:
Yeah, we'll have to make changes at release time anyhow (to adjust the repo), so IMO renaming the directory is simpler than looking for this line in `update.sh` (and then if there's a pre-release of 5.0.x later that we want to pull in, we can do that too).

Ok, I'll update.